docs/_docs/reference/changed-features/structural-types.md

```diff
@@ -66,38 +84,38 @@ The `person.name` and `person.age` selections are translated by
 the Scala compiler to:
 
 ```scala
-  person.selectDynamic("name").asInstanceOf[String]
-  person.selectDynamic("age").asInstanceOf[Int]
+person.selectDynamic("name").asInstanceOf[String]
+person.selectDynamic("age").asInstanceOf[Int]
 ```
 
 Besides `selectDynamic`, a `Selectable` class sometimes also defines a method `applyDynamic`. This can then be used to translate function calls of structural members. So, if `a` is an instance of `Selectable`, a structural call like `a.f(b, c)` would translate to
 
 ```scala
-  a.applyDynamic("f")(b, c)
+a.applyDynamic("f")(b, c)
 ```
 
 ## Using Java Reflection
 
-Structural types can also be accessed using [Java reflection](https://www.oracle.com/technical-resources/articles/java/javareflection.html). Example:
-
+If a structural type's base type does not have Selectable as a supertype, as below, it uses [Java reflection](https://www.oracle.com/technical-resources/articles/java/javareflection.html) instead:
```

Comment:
```suggestion
If a structural type's base type does not have `Selectable` as a supertype, it uses [Java reflection](https://www.oracle.com/technical-resources/articles/java/javareflection.html) instead.
If no base type is present, the base type is `Any`. 
**Note:** `Any` is not a subtype of `Selectable`, so base-type-less structural types will always use Java reflection.
Example:
```

Reply:
@sjrd / @bishabosha Is the above clear ?

Reply:
I think this implies that Java reflection is automatic - this is not the case, I think the original unchanged sentence is most clear